docs/source/quickstart.rst

```diff
@@ -1,2 +1,157 @@
 Quickstart
 ==========
+
+Welcome to Ignite quickstart guide that just gives essentials of getting a project up and running.
+
+In several lines you can get your model training and validating:
+
+Code
+----
+
+.. code-block:: python
+
+    from ignite.engines import Events, create_supervised_trainer, create_supervised_evaluator
+
+    model = Net()
+    train_loader, val_loader = get_data_loaders(train_batch_size, val_batch_size)
+    optimizer = torch.optim.SGD(model.parameters(), lr=0.01, momentum=0.8))
+    loss = torch.nn.NLLLoss()
+
+    trainer = create_supervised_trainer(model, optimizer, loss)
+    evaluator = create_supervised_evaluator(model,
+                                            metrics={
+                                                'accuracy': CategoricalAccuracy(),
+                                                'nll': Loss(loss)
+                                            })
+
+    @trainer.on(Events.ITERATION_COMPLETED)
+    def log_training_loss(engine):
```

Comment:
nit: I would rename this input arg to `trainer` to make it clear its this engine that's passed in